config/routes.rb

```diff
@@ -63,16 +63,13 @@
   resources :roles, only: [] do
     put :confirm, on: :member
   end
+  resources :mailings
```

Comment:
As I wrote for the `MailingsController`: if you we should not have any more actions then `show` and `index` for this resource, this should probably be done on router level:
```ruby
resources :mailings, only: [:index, :show]
```